roborl/ddpg/main.py

```diff
@@ -52,12 +53,13 @@ def run(args):
     parser.add_argument('--evaluate', type=int, default=10)
     parser.add_argument('--save_path', default=None)
     parser.add_argument('--save_every', type=int, default=10)
+    parser.add_argument('--exploration_policy', type=str, default='action')
```

Comment:
you don't need `type=str` as it is already a string and you could add `choices=['action', 'param']`